packages/node_modules/@webex/react-component-activity-share-files/src/styles.css

```diff
@@ -4,7 +4,7 @@
 }
 
 .shareList {
-  width: 320px;
+  max-width: 320px;
```

Comment:
Won't it feel inconsistent if the width of shared file A and shared file B is different based on the length of their names?